Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -73,10 +111,20 @@ So we have a Sprite, `happySunSprite` (or whatever you have named yours.) Let's
 to show it off. We'll start with creating a `Window` object for us to draw it onto. This can follow
 after the pervious lines in our Main() method.
 
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 Window SpriteWindow;
 spriteWindow = new Window("Learning Sprites!, 800, 800);
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+public static Window SplashKit.OpenWindow(string caption, int width, int height);
+public Window(string caption, int width, int height);
```

Comment:
C# code again, I think you want: 
```
Window spriteWindow;
spriteWindow = open_window("Learning Sprites!, 800, 800);
```
So that this matches the C# code, see here: (https://splashkit.io/api/windows/#open-window).